src/problems/euler/even_fibonacci_numbers/__main__.py

```diff
@@ -2,16 +2,16 @@
 
 from hamcrest import assert_that, equal_to
 
-from src.problems.euler.even_fibonacci_numbers.v1 import sum_fibonacci_evens
+from . import sum_fibonacci_evens
```

Comment:
So when we [implement it, we don't have to think about versions](https://github.com/percebus/algorithmic-excercises/pull/21#discussion_r1821451045)